monitoring/mock_uss/geoawareness/parsers/ed269/__init__.py

```diff
@@ -87,10 +88,28 @@ class WeekDateType(str, Enum):
     ANY = "ANY"
 
 
+class ED269TimeType(str):
+    """String that allows values which describe a time in ED-269 flavour of ISO 86001 format.
+
+    ED-269 standard specifies that a time instant type should be in the form of hh:mmS where S is
+    the timezone. However, examples are using the following format: 00:00:00.00Z
+    This class supports both formats.
+    """
+
+    def __new__(cls, value):
+        if isinstance(value, str):
+            t = arrow.get(value, ['HH:mm:ss.SZ', 'HH:mmZ']).timetz()
+        else:
+            t = value
+        str_value = str.__new__(cls, t.strftime('%H:%M:%S.%f')[:11] + t.strftime('%z').replace('+0000', 'Z'))
+        str_value.time = t
```

Comment:
Also something `implicitdict` doesn't do right now, but declaring this field in the class along with type-hinting would definitely help I think